### PR TITLE
script_output: log script for execution into log

### PR DESCRIPTION
**Pull request by asmorodskyi**
script_output: log script for execution into log

### CODE REVIEW

**Comment by asmorodskyi** [comment]
Do I need verification run for this ?

File: distribution.pm [hunk 1]
```diff
@@ -212,6 +212,7 @@ sub script_output {
         my $heretag = 'EOT_' . $marker;
         my $cat     = "cat > $script_path << '$heretag'; echo $marker-\$?-";
         testapi::wait_serial($self->{serial_term_prompt}, undef, 0, no_regex => 1);
+        bmwqemu::log_call("Content of $script_path : $cat");
```
Comment (foursixnine) [inline distribution.pm]: I'd vote for some markers at least, or enclose the $cat inside double quotes or double brackets or something in that fashion?

**Comment by foursixnine** [comment]
+1

File: distribution.pm (hunk 1 above)
Comment (asmorodskyi) [reply distribution.pm]:
```suggestion
        bmwqemu::log_call("Content of $script_path : \n \"$cat\" \n ");
```

something like this ?

Comment (foursixnine) [reply distribution.pm]: Somehow, yes :)

**Pull request merged**